src/v0/destinations/braze/util.js

```diff
@@ -634,6 +634,10 @@ function getPurchaseObjs(message) {
       parseInt(quantity, 10),
       timestamp,
     );
+    const extraProperties = _.omit(product, ['product_id', 'sku', 'price', 'quantity', 'currency']);
```

Comment:
can we define this list at one place & use it for mappings & for omitting here ?
so that any new field mapping requirement, change at one place would be sufficient.
this can go as part of different PR too

Reply:
Addressed